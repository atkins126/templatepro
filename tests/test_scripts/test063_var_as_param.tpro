{{if intvalue10|eq,intvalue10}}
intvalue10 = intvalue10
{{endif}}
x{{:value2|lpad,intvalue10}}x
