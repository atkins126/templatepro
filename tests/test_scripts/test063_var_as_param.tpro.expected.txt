intvalue10 = intvalue10
x  DANIELE2x
